### PR TITLE
feat: LT-1486 - Add Additional Details to End User Advisory

### PR DESCRIPTION
**Pull request by cstobart**

### CODE REVIEW

File: queries/end_user_advisories/serializers.py [hunk 1]
```diff
@@ -14,11 +15,24 @@ class EndUserAdvisorySerializer(serializers.ModelSerializer):
     end_user = EndUserSerializer()
     reasoning = serializers.CharField(required=False, allow_blank=True, allow_null=True, max_length=2000)
     note = serializers.CharField(required=False, allow_blank=True, allow_null=True, max_length=2000)
+    contact_email = serializers.EmailField()
     copy_of = serializers.PrimaryKeyRelatedField(queryset=EndUserAdvisoryQuery.objects.all(), required=False)
 
     class Meta:
         model = EndUserAdvisoryQuery
-        fields = ['id', 'end_user', 'reasoning', 'note', 'organisation', 'copy_of']
+        fields = ['id',
```
Comment (andytaylorkainos) [inline queries/end_user_advisories/serializers.py]: should fields not be a tuple?

File: queries/end_user_advisories/serializers.py [hunk 2]
```diff
@@ -32,6 +46,21 @@ def to_representation(self, value):
             }
         return repr_dict
 
+    def validate_nature_of_business(self, attrs):
```
Comment (andytaylorkainos) [inline queries/end_user_advisories/serializers.py]: not sure what `attrs` is?

File: queries/end_user_advisories/tests.py [hunk 3]
```diff
@@ -45,6 +47,11 @@ def test_view_end_user_advisory_query(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(response_data['note'], query.note)
         self.assertEqual(response_data['reasoning'], query.reasoning)
+        self.assertEqual(response_data['nature_of_business'], query.nature_of_business)
+        self.assertEqual(response_data['contact_name'], query.contact_name)
+        self.assertEqual(response_data['contact_email'], query.contact_email)
+        self.assertEqual(response_data['contact_telephone'], query.contact_telephone)
+        self.assertEqual(response_data['contact_job_title'], query.contact_job_title)
```
Comment (andytaylorkainos) [inline queries/end_user_advisories/tests.py]: 👍

File: queries/end_user_advisories/tests.py [hunk 4]
```diff
@@ -55,9 +62,9 @@ def test_view_end_user_advisory_query(self):
 
 
 class EndUserAdvisoryCreateTests(DataTestClient):
-
     url = reverse('queries:end_user_advisories:end_user_advisories')
 
+    @tag('only')
```
Comment (andytaylorkainos) [inline queries/end_user_advisories/tests.py]: tag

File: queries/end_user_advisories/tests.py [hunk 5]
```diff
@@ -149,3 +169,107 @@ def test_create_end_user_advisory_query_failure(self, end_user_type, name, websi
         response = self.client.post(self.url, data, **self.exporter_headers)
 
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_create_end_user_advisory_query_for_organisation_failure(self):
```
Comment (andytaylorkainos) [inline queries/end_user_advisories/tests.py]: Docstrings for tests please

File: queries/end_user_advisories/serializers.py (hunk 1 above)
Comment (cstobart) [reply queries/end_user_advisories/serializers.py]: it doesn't matter, it accepts either

File: queries/end_user_advisories/serializers.py (hunk 2 above)
Comment (cstobart) [reply queries/end_user_advisories/serializers.py]: built in parameter for overriding specific validation on fields

File: queries/end_user_advisories/serializers.py (hunk 1 above)
Comment (andytaylorkainos) [reply queries/end_user_advisories/serializers.py]: I know it takes both, but we've been using tuples consistently - can we stick to one approach? :)

Comment (cstobart) [reply queries/end_user_advisories/serializers.py]: 👍  done

**Pull request merged**